docs/src/docs/next-steps.md

```diff
@@ -1,11 +1,19 @@
 # Next Steps with Kit
 
 In this guide you'll learn how to:
+* Sign your ModelKit
 * Make your own Kitfile
 * The power of `unpack`
 * Read the Kitfile or manifest from a ModelKit
 * Tag ModelKits and keep your registry tidy
 
+## Signing your ModelKit
+
+Because ModelKits are OCI artifacts, they can be signed like any other OCI artifact (you may already sign your containers, for example.)
```

Comment:
Not a blocker, but having the period inside the parenthesis is making my eye twitch haha.